Plugins/Player/Player.cpp

```diff
@@ -5,14 +5,10 @@
 #include "API/CServerInfo.hpp"
 #include "API/CNWSArea.hpp"
 #include "API/CNWSPlayer.hpp"
-#include "API/CNWSMessage.hpp"
```

Comment:
I assume these lot were included transitively? I'd rather we include explicitly what we need than rely on transitive API includes that might change.

Reply:
Yeah, I'll restore em